php_phongo.h

```diff
@@ -155,7 +155,7 @@ void  php_phongo_prep_legacy_option_free(zval* options TSRMLS_DC);
 void php_phongo_read_preference_prep_tagsets(zval* tagSets TSRMLS_DC);
 bool php_phongo_read_preference_tags_are_valid(const bson_t* tags);
 
-void php_phongo_server_to_zval(zval* retval, mongoc_server_description_t* sd);
+bool php_phongo_server_to_zval(zval* retval, mongoc_server_description_t* sd);
```

Comment:
Make a point to consider other places that call `php_phongo_server_to_zval`. Those likely need logic to handle the exception. The server's `get_debug_info` handler is at least one instance.

Reply:
As per our analysis yesterday, the only other place this is called is in [`php_phongo_server_get_debug_info`](https://github.com/mongodb/mongo-php-driver/blob/fef008ed20ee55b87e1d55686c132d03b4950942/src/MongoDB/Server.c#L600), where the resulting zval is returned. The server description is properly freed, so there's no memory leaking there.